che-incubator/che-code
# Fix an automatic rebase against the upstream

closes https://github.com/eclipse/che/issues/21511

#### How to test
Run https://che-dogfooding.apps.che-dev.x6e0.p1.openshiftapps.com/#https://github.com/azatsarynnyy/java-spring-petclinic/tree/devfilev2.
Check that Che-Code is started.
